### PR TITLE
Allow sending messages with voice

### PR DESCRIPTION
**Pull request by windkomo**

### CODE REVIEW

File: src/components/WebChat/Main.jsx [hunk 1]
```diff
@@ -39,6 +40,21 @@ const Container = styled.div`
   }
 `;
 
+const C = styled.div``;
```
Comment (kevinadda) [inline src/components/WebChat/Main.jsx]: Why this name?

Comment (windkomo) [reply src/components/WebChat/Main.jsx]: forgot to remove it 😄

File: src/components/WebChat/index.jsx [hunk 2]
```diff
@@ -156,6 +159,25 @@ class WebChat extends React.Component {
...
+  };
+
+  async setTranscript(transcript) {
+    this.setState({
+      transcript,
+    });
+
+    if (transcript) {
+      await this.sendMessage(transcript);
+      this.setState({
```
Comment (kevinadda) [inline src/components/WebChat/index.jsx]: If transcript, you end up setting it twice in a row. It is necessary ?

File: src/components/WebChat/index.jsx [hunk 3]
```diff
@@ -164,21 +186,21 @@ class WebChat extends React.Component {
...
-  resetInput() {
-    this.setState({
-      input: undefined,
-    });
-  }
-
   handleKeyPress(e) {
     if (e && e.nativeEvent.keyCode === 13) {
-      this.sendMessage(this.state.input);
+      this.sendMessage();
```
Comment (kevinadda) [inline src/components/WebChat/index.jsx]: You remove the parameter hier, but line 180 you add the parameter to function signature. Is it intended?

File: src/components/WebChat/index.jsx [hunk 4]
```diff
@@ -164,21 +186,21 @@ class WebChat extends React.Component {
...
   }
 
-  resetInput() {
-    this.setState({
-      input: undefined,
-    });
-  }
-
   handleKeyPress(e) {
     if (e && e.nativeEvent.keyCode === 13) {
```
Comment (kevinadda) [inline src/components/WebChat/index.jsx]: Add a comment to say what is keyCode 13

File: src/components/message/Voice.jsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  render() {
+    const { isRecording } = this.props;
+
+    return (
+      <div>
+        {/* <button
+          onClick={() => {
+            annyang.trigger('salut');
+          }}
+        /> */}
```
Comment (kevinadda) [inline src/components/message/Voice.jsx]: remove?

File: src/components/WebChat/index.jsx (hunk 2 above)
Comment (windkomo) [reply src/components/WebChat/index.jsx]: No, you're right, it's old code I forgot to remove

File: src/components/WebChat/index.jsx (hunk 4 above)
Comment (windkomo) [reply src/components/WebChat/index.jsx]: 👍

**Pull request merged**